README.md

```diff
@@ -6,8 +6,15 @@ CiviForm aims to simplify the application process for benefits programs by re-us
 for multiple benefits applications. It is being developed by Google.org in collaboration with the
 City of Seattle.
 
-See the [latest end-to-end demo](https://youtu.be/glytMu8RTyA) (**NOTE**: This is a work in progress,
-so many screens are rough).
+Key features:
+
+- No-code administration: admins can add new questions and programs using the UI without the need for custom code
+- Multi-language support also through the admin UI
```

Comment:
s/also// ?  Or perhaps just "Multi-language support for both admins and citizens filling out the applications"?